File: prime.py
import time
import math

def main():
    n = 1000000
    numPrimes = 0
    start = time.clock()
    for i in range(1, n):
        if(isPrime(i)):
            numPrimes += 1
    end = time.clock()
    msec = (end - start) * 1000
    print("Found ", numPrimes, " primes in ", msec, "ms")

def isPrime(num):
    maxCheck = int(math.sqrt(num))
    for i in range(2, maxCheck):
        if(num % i == 0):
            return False
    return True

if __name__ == "__main__":
    main()

File: Prime.java
public class Prime {

    public static void main(String[] args) {
        int n = 100000;
        int numPrimes = 0;
        long start = System.currentTimeMillis();
        for(int i = 1; i < n; i++) {
            if(isPrime(i)) {
                numPrimes++;
            }
        }
        long end = System.currentTimeMillis();
        long difference = end - start;
        System.out.println("Found " + numPrimes + " primes in " + difference + "ms");
    }

    private static boolean isPrime(int n) {
        for(int i = 2; i < n / 2; i++) {
            if(n % i == 0) {
                return false;
            }
        }
        return true;
    }
}